featuretools_tsfresh_primitives/primitives/__init__.py

```diff
@@ -0,0 +1,71 @@
+from .autocorrelation import Autocorrelation
+from .binned_entropy import BinnedEntropy
+from .c3 import C3
+from .change_quantiles import ChangeQuantiles
+from .cid_ce import CidCe
+from .count_above_mean import CountAboveMean
+from .count_below_mean import CountBelowMean
+from .cwt_coefficients import CwtCoefficients
+from .energy_ratio_by_chunks import EnergyRatioByChunks
+from .fft_aggregated import FftAggregated
+from .fft_coefficient import FftCoefficient
+from .first_location_of_maximum import FirstLocationOfMaximum
+from .first_location_of_minimum import FirstLocationOfMinimum
+from .friedrich_coefficients import FriedrichCoefficients
+from .has_duplicate import HasDuplicate
+from .has_duplicate_max import HasDuplicateMax
+from .has_duplicate_min import HasDuplicateMin
+from .index_mass_quantile import IndexMassQuantile
+from .kurtosis import Kurtosis
+from .large_standard_deviation import LargeStandardDeviation
+from .last_location_of_maximum import LastLocationOfMaximum
+from .last_location_of_minimum import LastLocationOfMinimum
+from .length import Length
+from .linear_trend import LinearTrend
+from .longest_strike_above_mean import LongestStrikeAboveMean
+from .longest_strike_below_mean import LongestStrikeBelowMean
+from .max_langevin_fixed_point import MaxLangevinFixedPoint
+from .maximum import Maximum
+from .mean import Mean
+from .mean_abs_change import MeanAbsChange
+from .mean_change import MeanChange
+from .mean_second_derivative_central import MeanSecondDerivativeCentral
+from .median import Median
+from .minimum import Minimum
+from .number_crossing_m import NumberCrossingM
+from .number_cwt_peaks import NumberCwtPeaks
+from .number_peaks import NumberPeaks
+from .partial_autocorrelation import PartialAutocorrelation
+from .percentage_of_reoccurring_datapoints_to_all_datapoints import \
+    PercentageOfReoccurringDatapointsToAllDatapoints
+from .percentage_of_reoccurring_values_to_all_values import \
+    PercentageOfReoccurringValuesToAllValues
+from .quantile import Quantile
+from .range_count import RangeCount
+from .ratio_beyond_r_sigma import RatioBeyondRSigma
+from .ratio_value_number_to_time_series_length import \
+    RatioValueNumberToTimeSeriesLength
+from .sample_entropy import SampleEntropy
+from .skewness import Skewness
+from .spkt_welch_density import SpktWelchDensity
+from .standard_deviation import StandardDeviation
+from .sum_of_reoccurring_data_points import SumOfReoccurringDataPoints
+from .sum_of_reoccurring_values import SumOfReoccurringValues
+from .sum_values import SumValues
+from .symmetry_looking import SymmetryLooking
+from .time_reversal_asymmetry_statistic import TimeReversalAsymmetryStatistic
+from .value_count import ValueCount
+from .variance import Variance
+from .variance_larger_than_standard_deviation import \
+    VarianceLargerThanStandardDeviation
+
+PRIMITIVES_SUPPORTED = {value.name: value for key, value in locals().items() if isinstance(value, type)}
```

Comment:
let's call this `SUPPORTED_PRIMITIVES` and use `locals().values()` since we aren't using key here

Reply:
And make it a list instead of a dictionary.  We can get the name from the value, if we need it